File: src/main/java/com/gmail/borlandlp/minigamesdtools/lobby/LobbyCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.lobby;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class LobbyCreatorHub extends CreatorHub {
    public ServerLobby createLobby(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (ServerLobby) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/lobby/LobbyCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.lobby

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class LobbyCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): ServerLobby {
        return super.create(itemID, dataProvider) as ServerLobby
    }
}